docs/topics/mta-rn-resolved-issues-6-2-0.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes-6.2/master.adoc
+
+:_content-type: REFERENCE
+[id="mta-rn-resolved-issues-6-2-0_{context}"]
+= Resolved issues
+
+The following highlighted issues have been resolved in MTA version 6.2.0.
+
+.Analysis wizard
+The release of MTA 6.2.0 resolves the issue that Analysis wizard was stuck on the custom rules page on moving *Back* from the Repository tab.
+For more information on this issue, see link:https://issues.redhat.com/browse/MTA-464[MTA-464].
+
+.Tags & Reports table
```

Comment:
Should be "Tags & Reports tabs" (not table)

Reply:
@anarnold97 those are 2 different tabs
![image-2023-04-19-11-34-36-844](https://github.com/windup/windup-documentation/assets/60384172/6fac86bf-af32-496d-9899-13602ac23962)